atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/TransactionScopedCacheImpl.java

```diff
@@ -87,11 +101,52 @@ public synchronized void delete(TableReference tableReference, Set<Cell> cells)
         } else {
             return Futures.transform(
                     valueLoader.apply(tableReference, cacheLookup.missedCells()),
-                    remoteReadValues -> processRemoteRead(tableReference, cacheLookup, remoteReadValues),
+                    remoteReadValues -> processRemoteRead(
+                            tableReference, cacheLookup.cacheHits(), cacheLookup.missedCells(), remoteReadValues),
                     MoreExecutors.directExecutor());
         }
     }
 
+    @Override
+    public synchronized NavigableMap<byte[], RowResult<byte[]>> getRows(
+            TableReference tableRef,
+            Iterable<byte[]> rows,
+            ColumnSelection columnSelection,
+            Function<Set<Cell>, Map<Cell, byte[]>> cellLoader,
+            Function<Iterable<byte[]>, NavigableMap<byte[], RowResult<byte[]>>> rowLoader) {
```

Comment:
Not for this PR - The signature makes sense, we should look at modifying signatures of other apis in this class as well for uniformity, in particular, not use BiFunction.

Reply:
👍